reference/toolstacks/nodejs/README.md

```diff
@@ -20,59 +20,70 @@ To use Platform.sh and Node.js together, configure the ``.platform.app.yaml``
 file with a few key settings, as described here (a complete example is included
 at the end).
 
-1. Specify the `type` as `nodejs` you should definitely specify a version for example `type: nodejs:4.2`
+1. Specify the `type` as `nodejs`. You may also specify a version for example `type: nodejs:4.4`.
```

Comment:
As for (2), it could be argued that we want to go with 4.x since it's the officially stable version.